site/includes/footerresources.hbs

```diff
@@ -1,5 +1,6 @@
 <!--[if gte IE 9 | !IE ]><!-->
 <script src="{{#if offline}}{{assets}}/../wet-boew/js{{else}}https://ajax.googleapis.com/ajax/libs{{/if}}/jquery/{{environment.jqueryVersion}}/jquery.js"></script>
+<script>window.jQuery || document.write('<script src="{{assets}}/../wet-boew/js/jquery/{{environment.jqueryVersion}}/jquery.js"><\/script>');</script>
```

Comment:
@nschonni are you ok with this now?
I will be starting the release at 9:00 and wondering if this should be merged?
@LaurentGoderre @duboisp any comments?